studio/pages/new/[slug].tsx

```diff
@@ -20,40 +27,99 @@ import {
   PASSWORD_STRENGTH,
   PASSWORD_STRENGTH_COLOR,
   PASSWORD_STRENGTH_PERCENTAGE,
+  PRICING_PLANS,
+  PRICING_PLANS_DEFAULT,
 } from 'lib/constants'
 
 import { useStore, withAuth } from 'hooks'
+import { IRootStore } from 'stores'
 import { WizardLayout } from 'components/layouts'
 import FormField from 'components/to-be-cleaned/forms/FormField'
 import Panel from 'components/to-be-cleaned/Panel'
 import InformationBox from 'components/ui/InformationBox'
+import { Organization } from 'types'
+import { getURL } from 'lib/helpers'
+
+interface StripeCustomer {
+  paymentMethods: any
+  customer: any
+  error?: any
+}
 
 interface IHomePageStore {
-  store: any
-  projectOrganizations: Dictionary<any>[]
+  store: IRootStore
+  organizations: Dictionary<any>[]
+  currentOrg: Organization | undefined
+  isEmptyOrganizations: boolean
+  isEmptyPaymentMethod: boolean | undefined
+  isInvalidSlug: boolean
+  isOverFreeProjectLimit: boolean
+  stripeCustomerId: string | undefined
+  loadStripeAccount: () => void
 }
 class HomePageStore implements IHomePageStore {
```

Comment:
on a separate note - we're planning to move away from local mobx stores right? thinking if we can just refactor it now then for this page - we can just use the data from the useStore hook